api/schema/aws_account_resource.go

```diff
@@ -53,9 +53,9 @@ var (
 					Description: "Display name for the AWS account.",
 					Required:    true,
 				},
-				"organization_master_account_id": StringResourceAttributeWithMode{
```

Comment:
if we are going to support the org in future having organization_master_account_id along with org_id makes it easier. can we add it.

Reply:
if not required, everything else LGTM

Reply:
When we support orgs later, we shouldn't use the master account ID to group by. We should use the org ID directly. By the time we get to supporting orgs, the backend will have changed to group by org ID.